character-fx.js

```diff
@@ -234,8 +234,9 @@ class CharacterFx {
 
     this.lastSSS = isSSS;
     const _updateSonicBoomMesh = () => {
-      if (this.player.isLocalPlayer && !this.sonicBoom) {
+      if ( !this.sonicBoom  && !this.player.isNpcPlayer ) {
         this.sonicBoom = metaversefile.createApp();
+        this.sonicBoom.player = this.player;

```

Comment:
make player id a component, if its null player then is local player, otherwise look itup